rust/src/cells/methods.rs

```diff
@@ -62,7 +62,6 @@ impl NcCell {
     ///
     /// - Breaks the UTF-8 string in `gcluster` down, setting up this NcCell.
     /// - Returns the number of bytes copied out of `gcluster`.
-    /// - The styling of the cell is left untouched, but any resources are released.
```

Comment:
@joseluis wrot ethis, but it looks like this is just calling through to `cell_prime()` from the C library, in which case the style is indeed changed, but the resources are released. so i'd keep the latter half of this comment.

Reply:
Ok!  Tbh I don't understand what resources it's referring to (I only started playing with this library today), but the styling bit looked obviously wrong.

Reply:
Essentially, a `cell` is a fixed-sized structure (16 bytes, to be precise), and thus has only 4 bytes available for the Unicode EGC it contains. That's enough for the vast majority of EGCs, but not all. If your EGC encodes to more than 4 UTF-8 bytes, it's spilled out into the associated Plane's EGCPool (this is why a Plane is involved in just about every call involving a cell). If you replace the EGC, we need to free up the EGCPool entry. just fyi =]

Reply:
Ah right, so it's resources for an existing EGC you're replacing.  Got it, thanks :)

Reply:
exactly. btw since you're working with rust, let me point you at the c library's manual pages, e.g. https://notcurses.com/notcurses_cell.3.html; they're all available at https://notcurses.com/. if you see anything that's underdocumented, don't hesitate to file a bug or a PR!